aws/src/test/java/org/apache/iceberg/aws/TestAwsClientFactories.java

```diff
@@ -132,6 +136,41 @@ public void testLakeFormationAwsClientFactorySerializable() throws IOException {
         .isInstanceOf(LakeFormationAwsClientFactory.class);
   }
 
+  @Test
+  public void testDefaultAwsClientFactoryWithCredentialsProvider() {
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put(AwsProperties.AWS_REGION, Region.AWS_GLOBAL.toString());
```

Comment:
ah interesting I did not know this AWS_GLOBAL region thing, does that work for S3 access so I don't need to know the region of the bucket?

Reply:
AWS_GLOBAL I just used it for junit testing